webserver/Migrations/20231003160237_REmodeling.Designer.cs

```diff
@@ -241,6 +241,18 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
                     b.Property<decimal>("Price")
                         .HasColumnType("decimal(18,2)");
 
+                    b.Property<int>("area")
+                        .HasColumnType("int");
```

Comment:
This is a float. Example: `77,3 square meters`

Reply:
I usually only see R.Es areas' being marketed with integer numbers so that's what i went with